backend/core/src/listings/combined/combined-listing.entity.ts

```diff
@@ -134,10 +146,43 @@ ON l.utilities_id = util.id
 LEFT JOIN "address" addr
 ON l.building_address_id = addr.id
 
+-- Some columns representing numeric data use the "text" type
+-- We need to convert those to numbers so we can filter on them correctly
+-- There is no input validation on these, so we need to validate format before converting
+-- In a perfect world we'd write a function, but that's impractical here compared to a one-liner
+
+-- Breaking down the type conversion logic
+-- CASE WHEN column ~ '^[0-9]+\\.{0,1}[0-9]+$' THEN TO_NUMBER(column, '99999') ELSE 0 END
+-- "if text column matches "(num)+.(num)+", then convert it to a number with up to 5 digits, otherwise use the number 0
+-- examples of matches: "123", "12345", "12345.6", "12345.67"
+-- examples of non-matches: "", " ", "Contact listing agent", "12.", "12.34.5", ".12", "1,234",  "1,234.0"
```

Comment:
(non-blocking, suggestion) You might have mentioned this in your msg earlier but one thing we could do here is store both the raw text version of these numeric fields, as well as our "best guess approximation" of what the number is and store as an int/float - we can be a little more liberal here and do stuff like ignore "." and "," and use in the search filtering, while still using the raw text fields for the UI.

Reply:
We could totally do that.  I was even thinking of putting filter fields into a single jsonb field to minimize the amount of work adding and removing them, but I decided against it this go around.  We can still index and query against them, but it would just be in the format `filter->>'maxBedrooms'` instead.  Adding the original string values next to them for visual debugging would be pretty straightforward.  Even if we don't combine the converted fields here into a single field, we could still combine the original values into a their own field to keep from blowing the table schema up too much.  

That's my long winded way of saying "yes, and..."